en/docs/reference/connectors/kafka-connector/kafka-connector-avro-producer-example.md

```diff
@@ -18,51 +18,180 @@ Follow these steps to set up the Integration Project and the Connector Exporter
-                     <valueSerializerClass>io.confluent.kafka.serializers.KafkaAvroSerializer</valueSerializerClass>
-                     <schemaRegistryUrl>http://localhost:8081</schemaRegistryUrl>
-                     <maxPoolSize>100</maxPoolSize>
-                </kafkaTransport.init>
-                <kafkaTransport.publishMessages>
-                     <topic>{$ctx:topic}</topic>
-                     <key>{$ctx:key}</key>
-                     <value>{$ctx:value}</value>
-                     <valueSchema>{$ctx:valueSchema}</valueSchema>
-                </kafkaTransport.publishMessages>
-            </inSequence>
-            <outSequence/>
-            <faultSequence/>
-        </resource>
-    </api>
-    ```
+1. Specify the API name as `KafkaTransport` and API context as `/publishMessages`.
+
+2. To configure the resource click on the API Resource and go to **Properties** view. Select the method POST.
+
+3. Drag and drop the property mediator from the mediator palette to the request path of the API resource.
+
+4. Specify values for the property mediator:
+   <table>
+      <tr>
+         <th>Parameter</th>
+         <th>Value</th>
+      </tr>
+
+      <tr>
+        <td>New Property Name</td>
+        <td>valueSchema</td>
+      </tr>
+
+      <tr>
+        <td>Property Data Type</td>
+        <td>STRING</td>
+      </tr>
+
+      <tr>
+        <td>Property Action</td>
+        <td>set</td>
+      </tr>
+
+      <tr>
+        <td>Property Scope</td>
+        <td>default</td>
+      </tr>
+
+      <tr>
+        <td>Value</td>
+        <td>json-eval($.test)</td>
+      </tr>
+    </table>
+
+5. Similarly, drag and drop 3 more property mediators from the mediator palette to the request path of the API resource and specify values as follows.
+   <table>
+      <tr>
+         <th>Property Name</th>
+         <th>Value</th>
+      </tr>
+
+      <tr>
+        <td>value</td>
+        <td>json-eval($.value)</td>
+      </tr>
+
+      <tr>
+        <td>key</td>
+        <td>json-eval($.key)</td>
+      </tr>
+
+      <tr>
+        <td>topic</td>
+        <td>json-eval($.topic)</td>
+      </tr>
+    </table>
+   
+6. Next drag and drop the 'publishMessages' operation of the KafkaTransport Connector to the Design View as shown below.
+   <a href="{{base_path}}/assets/img/integrate/connectors/kafka"><img src="{{base_path}}/assets/img/integrate/connectors/kafka/kafka-avro-example.png" title="Kafka Avro Example" width="800" alt="Kafka Avro Example"/></a>
+
+7. Create a connection from the properties window by clicking on the '+' icon as shown below.
+
+   <a href="{{base_path}}/assets/img/integrate/connectors/kafka"><img src="{{base_path}}/assets/img/integrate/connectors/kafka/kafka-add-connection.png" title="Create Connection" width="800" alt="Create Connection"/></a>
+
+   In the popup window, provide the following parameters and click **Finish**.
+
+   <table>
```

Comment:
This table is not getting rendered properly

Reply:
Fixed with 1889736aed7fbdd77eb868c72dfc10b8f7049056